requirements.txt

```diff
@@ -1,4 +1,4 @@
-aniso8601==8.0.0
+aniso8601==7.0.0
```

Comment:
interesting that you had a manually change that b/c we pulled in 7 in a previous PR if I recall.

Reply:
Yeah, I'm not sure how/when it was reset, but it was a quick fix so 🤷